presto-verifier/src/main/java/com/facebook/presto/verifier/source/VerifierDao.java

```diff
@@ -34,32 +35,40 @@ public interface VerifierDao
             "  control_catalog varchar(256) NOT NULL,\n" +
             "  control_schema varchar(256) NOT NULL,\n" +
             "  control_query text NOT NULL,\n" +
+            "  control_query_id mediumtext DEFAULT NULL,\n" +
```

Comment:
Strange, didn't we put control and test query Ids into the table before this change?
The ones that you click on, like "Control Query ID".

Reply:
Yeah, the "Control Query ID" link on UI is populated with the query_id returned by the actual control query run triggered by Verifier today. If we reuse the shadow/prod query output and don't trigger the control/test query runs, we have to pass the query_ids to Verifer through xdb, so that the UI can display them.

Otherwise our debugging experience is bad. Found it out from testing.